src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -281,11 +281,11 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
         <DestructibleLayout />
       </>
     );
-    expect(wrapper.find('[data-testid="local-breadcrumbs"]')!.getElement()).toBeEmptyDOMElement();
+    expect(wrapper.find('[data-testid="local-breadcrumbs"]')!.findBreadcrumbGroup()).toBeFalsy();
```

Comment:
New behavior here – instead of rendering nothing we render breadcrumbs placeholder to be discovered by our funnel metrics collector